crates/fs/public/directory.rs

```diff
@@ -237,6 +236,11 @@ impl PublicDirectory {
         }
     }
 
+    /// Creates a new directory using the given metadata.
+    pub fn with_rc(time: DateTime<Utc>) -> Rc<Self> {
+        Rc::new(Self::new(time))
+    }
+
```

Comment:
Sorry, I may sound harsh, but do we need this? This saves ~5 characters on call sites compared to `Rc::new(PublicDirectory::new(...))` plus a set of parenthesis. As far as I can tell we only use it once.

Or is `with_rc` possibly a common idiom in Rust that I don't know about?

Reply:
Let's get rid of it. I thought I would need it a lot.